src/Craned/Craned/JobManager.cpp

```
@@ -0,0 +1,924 @@
+      });
+
+  // Task Status Change Event
+  m_task_status_change_async_handle_ =
+      m_uvw_loop_->resource<uvw::async_handle>();
+  m_task_status_change_async_handle_->on<uvw::async_event>(
+      [this](const uvw::async_event&, uvw::async_handle&) {
+        EvCleanTaskStatusChangeQueueCb_();
+      });
+
+  m_change_task_time_limit_async_handle_ =
+      m_uvw_loop_->resource<uvw::async_handle>();
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)